src/main/resources/templates/fragments/layout.html

```diff
@@ -2,7 +2,10 @@
 <html th:fragment="layout (template, viewName)" xmlns:th="http://www.w3.org/1999/xhtml">
 <head>
     <title>Gateway Java Sample</title>
-    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.2/css/bootstrap.min.css" integrity="sha384-PsH8R72JQ3SOdhVi3uxftmaW6Vc51MKb0q5P2rRUpPvrszuE4W1povHYgTpBfshb" crossorigin="anonymous">2
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.2/css/bootstrap.min.css" integrity="sha384-PsH8R72JQ3SOdhVi3uxftmaW6Vc51MKb0q5P2rRUpPvrszuE4W1povHYgTpBfshb" crossorigin="anonymous">
```

Comment:
Minor thing.. you probably want it to use `//` opposed to scoping it to https://

Reply:
Done

https://github.com/Mastercard-Gateway/gateway-java-sample-code/pull/5/commits/b1a345b3dc3e50262475daa8ffcbcf8c28b09352